### PR TITLE
Add app

### PR DESCRIPTION
**Pull request by jarqvi**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    build: .
+    ports:
+      - 8080:8080
+    volumes:
+      - ./spec:/usr/share/nginx/html/spec
+    environment:
+      URLS: "[
+          { url: '/spec/dns.yaml', name: 'DNS' },
+          { url: '/spec/app.yaml', name: 'APP' }, 
+          { url: '/spec/db.yaml', name: 'DATABASE' }
```
Comment (mhemrg) [inline docker-compose.yml]:
```suggestion
          { url: '/spec/paas.yaml', name: 'PaaS' }, 
          { url: '/spec/dbaas.yaml', name: 'DBaaS' }
```

File: spec/app.yaml [hunk 2]
```diff
@@ -0,0 +1,1196 @@
+swagger: '2.0'
+info:
+  description: 'Manage the created apps'
```
Comment (mhemrg) [inline spec/app.yaml]:
```suggestion
  description: 'Manage your apps using our API'
```

File: spec/app.yaml [hunk 3]
```diff
@@ -0,0 +1,1196 @@
...
+  version: '1.0.0'
+  title: 'APP'
```
Comment (mhemrg) [inline spec/app.yaml]:
```suggestion
  title: 'PaaS'
```

**Comment by mhemrg** [comment]
Awesome, you've done a great job man! I've had a few minor suggestions.

**Comment by jarqvi** [comment]
Done.

**Pull request merged**